wgpu-types/src/lib.rs

```diff
@@ -5505,8 +5505,10 @@ bitflags::bitflags! {
     #[repr(transparent)]
     #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
     #[cfg_attr(feature = "serde", serde(transparent))]
-    #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+    #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Default)]
     pub struct TextureUsages: u32 {
+        /// For use in texture views, expands to the full set of usage flags of the texture
+        const FROM_PARENT = 0;
```

Comment:
I'm torn here, on the one hand `= 0` bitflags behave really weird, but this is also a way to explicitly label the "none" use case. I think this is fine?